larpandora/LArPandoraInterface/LArPandoraGeometry.h

```diff
@@ -93,25 +93,69 @@ namespace lar_pandora {
     /**
      *  @brief  Constructor
      *
-     *  @param  cryostat  the cryostat ID
-     *  @param  tpc       the tpc ID
+     *  @param  cryostat         the cryostat ID
+     *  @param  tpc              the tpc ID
+     *  @param  centerX          centre of tpc volume (X)
+     *  @param  centerY          centre of tpc volume (Y)
+     *  @param  centerZ          centre of tpc volume (Z)
+     *  @param  widthX           width of tpc volume (X)
+     *  @param  widthY           width of tpc volume (Y)
+     *  @param  widthZ           width of tpc volume (Z)
      */
-    LArDaughterDriftVolume(const unsigned int cryostat, const unsigned int tpc);
+    LArDaughterDriftVolume(const unsigned int cryostat, const unsigned int tpc,
+                           const float centerX, const float centerY, const float centerZ,
+                           const float widthX, const float widthY, const float widthZ);
 
     /**
-     *  @brief  return cryostat ID
+     *  @brief  Return cryostat ID
      */
     unsigned int GetCryostat() const;
 
     /**
-     *  @brief  return tpc ID
+     *  @brief  Return tpc ID
      */
     unsigned int GetTpc() const;
 
-  private:
-    unsigned int m_cryostat;
-    unsigned int m_tpc;
-  };
+    /**
+     *  @brief  Return X position at centre of tpc volume
+     */
+    float GetCenterX() const;
+
+    /**
+     *  @brief  Return Y position at centre of tpc volume
+     */
+    float GetCenterY() const;
+
+    /**
+     *  @brief  Return Z position at centre of tpc volume
+     */
+    float GetCenterZ() const;
+
+    /**
+     *  @brief  Return X span of tpc volume
+     */
+    float GetWidthX() const;
+
+    /**
+     *  @brief  Return Y span of tpc volume
+     */
+    float GetWidthY() const;
+
+    /**
+     *  @brief  Return Z span of tpc volume
+     */
+    float GetWidthZ() const;
+
+private:
+    unsigned int    m_cryostat;
+    unsigned int    m_tpc;
+    float           m_centerX;
+    float           m_centerY;
+    float           m_centerZ;
+    float           m_widthX;
+    float           m_widthY;
+    float           m_widthZ;
```

Comment:
Add doxygen style comments `///< Description of variable`

Reply:
I see no such description on other Class's private variables however Getblabla and constructor functions all have their parameters properly written. I mimicked the rest of the file for these.